packages/atomic/src/components/insight/atomic-insight-search-box/atomic-insight-search-box.tsx

```diff
@@ -0,0 +1,86 @@
+import {Component, h, Prop, State} from '@stencil/core';
+import {buildInsightSearchBox} from '@coveo/headless/insight';
+import {SearchBox, SearchBoxState} from '@coveo/headless';
```

Comment:
These types should also be coming from `'@coveo/headless/insight'` I think 🤔 

If they are missing it means it's a bug from Headless package.

Reply:
yeah I couldn't find it there.. I did add the missing import for `buildInsightSearchBox` and now see that there is an `InsightSearchBox` too that I can export! 
will tag the headless guys to have a look too!!